mkShapesRDF/shapeAnalysis/mkShapesRDF.py

```diff
@@ -279,6 +294,14 @@ def main():
         Warning in <Snapshot>: A lazy Snapshot action was booked but never triggered.
         cling::DynamicLibraryManager::loadLibrary(): libOpenGL.so.0: cannot open shared object file: No such file or directory
         Error in <AutoloadLibraryMU>: Failed to load library /cvmfs/sft.cern.ch/lcg/releases/ROOT/6.28.00
+        TClass::Init:0: RuntimeWarning: no dictionary for class edm::Hash<1> is available
+        TClass::Init:0: RuntimeWarning: no dictionary for class edm::ParameterSetBlob is available
+        TClass::Init:0: RuntimeWarning: no dictionary for class edm::ProcessHistory is available
+        TClass::Init:0: RuntimeWarning: no dictionary for class edm::ProcessConfiguration is available
+        TClass::Init:0: RuntimeWarning: no dictionary for class pair<edm::Hash<1>,edm::ParameterSetBlob> is available         
+        cling::DynamicLibraryManager::loadLibrary(): libGLU.so.1: cannot open shared object file: No such file or directory
+        Error in <TNetXNGFile::Close>:
+        Warning in <TChain::AddFile>: Adding tree with no entries from file
         """
```

Comment:
maybe we can also add the warning below among strings that are classified as "normal errors"
`Info in <ACLiC>: unmodified script has already been compiled and loaded`